src/assert.js

```diff
@@ -0,0 +1,29 @@
+function Assert() {
+
+};
+
+Assert.IsString = function(param) {
+    if (param) {
+        if (typeof param !== 'string') {
+            throw new TypeError("Parameter should be string not a " + typeof param);
+        }
+    }
+}
+
+Assert.isNumber = function(param) {
+    if (param) {
```

Comment:
*NEVER* use strings, numbers as conditions in if (where true or false is expected). Although it is not a syntax error, it very bad practice, since it is a recipe for bugs.

One of the examples with numbers is that 0 is considered as false, while everything else true. So here is a bug here, if I call:
```
Assert.isNumber(0);
```
then
```
if (0) { /* if body */ }
```
Then if body wont be executed, since it means if (false).